foundry.toml

```diff
@@ -2,6 +2,6 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
-remappings = ['openzeppelin-contracts/=lib/openzeppelin-contracts/contracts/']
+remappings = ['@openzeppelin/=lib/openzeppelin-contracts/', 'forge-std/=lib/forge-std/src/', 'ds-test/=lib/forge-std/lib/ds-test/src/']
```

Comment:
Why are we remapping ds-test here?

Reply:
Could have sworn that there were resulting in errors. I'm not seeing them anymore. Will revert changes here. Thanks for the catch.